### PR TITLE
Create exception namespace

### PR DESCRIPTION
**Pull request by apope1**
Ba... Pnm... Facui si exceptiile si bagai si diagrama de clasa.

### CODE REVIEW

File: App/App/Exception/InvalidEmailAddressException.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace App.Exception
```
Comment (ovidiuenache) [inline App/App/Exception/InvalidEmailAddressException.cs]: Why have an InvalidCredentialsException and also have InvalidEmailAddressException?

File: App/App/Exception/InvalidPasswordException.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace App.Exception
+{
+    public class InvalidPasswordException : BaseException
```
Comment (ovidiuenache) [inline App/App/Exception/InvalidPasswordException.cs]: Why have an InvalidCredentialsException and also have InvalidPasswordException?

File: App/App/Exception/InvalidEmailAddressException.cs [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+{
+    public class InvalidEmailAddressException : BaseException
+    {
+        public InvalidEmailAddressException(string message) : base(message) { }
+    }
+}
```
Comment (ovidiuenache) [inline App/App/Exception/InvalidEmailAddressException.cs]: Why have an InvalidCredentialsException and also have InvalidEmailAddressException?

File: App/App/Exception/InvalidPasswordException.cs [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+    {
+        public InvalidPasswordException(string message) : base(message) { }
+    }
+}
```
Comment (ovidiuenache) [inline App/App/Exception/InvalidPasswordException.cs]: Why have an InvalidCredentialsException and also have InvalidPasswordException?

File: App/App/Exception/InvalidEmailAddressException.cs (hunk 3 above)
Comment (apope1) [inline App/App/Exception/InvalidEmailAddressException.cs]: InvalidEmailAddressException is used in the Registration part, while InvalidCredentialsException is used when Logging In. This is thrown when you type an incorrect email address while registering (e. g. it doesn't contain @example.org or something like that).

File: App/App/Exception/InvalidPasswordException.cs (hunk 4 above)
Comment (apope1) [inline App/App/Exception/InvalidPasswordException.cs]: The same goes here. InvalidCredentialsException is used when Logging In, while InvalidPasswordException is thrown when creating a new account. This is used for validating the new password (e. g. Must be at least 8 characters long. Must contain a number, an uppercase character and a lowercase character, etc.).

**Pull request merged**